scripts/cheatsheet.txt

```diff
@@ -0,0 +1,18 @@
+import sentencepiece as spm 
```

Comment:
It would be helpful to have a comment on top of this script to know what's it's general purpose.